### PR TITLE
performance improvements

### PR DESCRIPTION
**Pull request by raffaelespazzoli**
bump to utils 1.3.2
changed api to allow for an array of patches to improve perf
fixed issue with logging

Signed-off-by: raffaelespazzoli <raffaele.spazzoli@gmail.com>

### CODE REVIEW

File: config/local-development/tilt/replace-image.yaml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 - op: replace
   path: /spec/template/spec/containers/1/image
   value:
-    quay.io/raffaelespazzoli/patch-operator:latest
\ No newline at end of file
+    quay.io/raffaelespazzoli/patch-operator:latest
```
Comment (sabre1041) [inline config/local-development/tilt/replace-image.yaml:4]: This should point to `redhat-cop` org

**Review by sabre1041 — CHANGES_REQUESTED** [review]
Small change

File: config/local-development/tilt/replace-image.yaml (hunk 1 above)
Comment (raffaelespazzoli) [reply config/local-development/tilt/replace-image.yaml:4]: no this is for local development, it should stay that way

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**